services/gateway/apis/http-version2/swagger/parameters/multisigTransactions.json

```diff
@@ -116,5 +116,49 @@
                 }
             }
         }
+    },
+    "patchMultisigTransaction": {
+        "type": "object",
+        "in": "body",
+        "schema": {
+            "$ref": "#/definitions/patchMultisigTransaction"
+        },
+        "required": [
```

Comment:
Those lines (126-162) are not needed in parameters. The definition for `patchMultisigTransaction`  can be moved to `multisigTransactions` definitions (services/gateway/apis/http-version2/swagger/definitions/multisigTransactions.json).